src/controllers/authController.ts

```diff
@@ -1,63 +1,52 @@
-import * as google from 'googleapis';
-import * as url from 'url';
+import * as passport from 'passport';
+import { OAuth2Strategy } from 'passport-google-oauth';
 import { Request, Response, NextFunction } from 'express';
 
-const { OAuth2 } = google.auth;
+import { User } from '../models';
 
-let callbackURL = 'http://localhost:3000/auth/google/callback';
-if (process.env.PRODUCTION == 'true') {
-  callbackURL = 'http://sources.dailybruin.com/auth/google/callback';
-}
+const callbackURL =
+  process.env.PRODUCTION === 'true'
+    ? 'http://sources.dailybruin.com/auth/google/callback'
```

Comment:
ideally we would set another env (and header validations + cors) to set this URL, but this can be a TBD

Reply:
Agreed! I'm not super familiar with headers and CORS, but could you make a PR so we don't forget this?